src/cmd-kola

```diff
@@ -58,33 +54,8 @@ subargs = args.subargs or [default_cmd]
 kolaargs.extend(subargs)
 kolaargs.extend(unknown_args)
 
-if args.basic_qemu_scenarios:
-    if arch == "x86_64":
-        os.mkdir(outputdir)  # Create the toplevel output dir
-        for scenario in BASIC_SCENARIOS:
-            kolaargs.extend(['--output-dir',
-                             os.path.join(outputdir, scenario.replace('=', '-'))])
-            subargs = kolaargs + ['--qemu-' + scenario, 'basic']
-            print(subprocess.list2cmdline(subargs), flush=True)
-            subprocess.check_call(subargs)
-        if not args.skip_secure_boot:
-            for scenario in BASIC_SCENARIOS_SECURE_BOOT:
-                kolaargs.extend(['--output-dir',
-                                 os.path.join(outputdir, scenario.replace('=', '-'))])
-                # See https://issues.redhat.com/browse/COS-2000 - there's
-                # some bug with shim/grub2 that fails with secure boot on < ~1300MiB of RAM.
-                # But we're not going to block on that; real world OCP worker nodes are at least 16GiB etc.
-                subargs = kolaargs + ['--qemu-' + scenario, 'basic'] + ["--qemu-memory", "1536"]
-                print(subprocess.list2cmdline(subargs), flush=True)
-                subprocess.check_call(subargs)
-    else:
-        # Basic qemu scenarios using nvme and uefi
-        # are not supported on multi-arch
```

Comment:
This original logic dictated which basic test would run on which architecture. Will we have the same control over the proper kola tests?

Reply:
Yeah we do; Kola tests proper have a field which can be set for this `Architectures` currently its left null; and from my understanding of the kola test [harness](https://github.com/coreos/coreos-assembler/blob/0d4a48566e1e80beb57e2435403d8228c3e43324/mantle/kola/harness.go#L602) it will take that as allowing these tests to run all all architectures. If there is one or two we want to ignore we can set the field `ExcludeArchitectures` as well.

Reply:
Perfect, that should work nicely.